fix TestISequencer_MultipleActualizes()

https://github.com/voedger/voedger/actions/runs/14567751840
```
  --- FAIL: TestLongRace (172.94s)
      isequencer_test.go:778: 
          	Error Trace:	/home/runner/work/voedger/voedger/pkg/isequencer/isequencer_test.go:778
          	Error:      	Not equal: 
          	            	expected: 0x8d
          	            	actual  : 0x8e
          	Test:       	TestLongRace
          	Messages:   	Sequence number should not be incremented after Actualize
```

--- a/pkg/isequencer/impl.go
+++ b/pkg/isequencer/impl.go
@@ -79,6 +79,10 @@ func (s *sequencer) startActualizer() {
 func (s *sequencer) stopFlusher() {
 	s.flusherCtxCancel()
 	s.flusherWG.Wait()
+	select {
+	case <-s.flusherSig:
+	default:
+	}
 }
 
 // signalToFlushing is used to signal the flusher to start flushing.
@@ -440,7 +444,7 @@ func (s *sequencer) checkSequencingTransactionInProgress() {
 func (s *sequencer) Actualize() {
 	// Validate Sequencing Transaction status
 	s.checkSequencingTransactionInProgress()
-	
+
 	// - Validate Actualization status
 	// Set s.actualizerInProgress to true
 	if !s.actualizerInProgress.CompareAndSwap(false, true) {

--- a/pkg/isequencer/isequencer_test.go
+++ b/pkg/isequencer/isequencer_test.go
@@ -692,6 +692,41 @@ func TestISequencer_Actualize(t *testing.T) {
 		})
 	})
 
+	t.Run("basic usage", func(t *testing.T) {
+		storage := createDefaultStorage()
+		storage.SetPLog(map[isequencer.PLogOffset][]isequencer.SeqValue{})
+		params := createDefaultParams()
+		seq, cleanup := isequencer.New(params, storage, mockedTime)
+		defer cleanup()
+		initialOffset := isequencer.WaitForStart(t, seq, 1, 1, true)
+
+		num, err := seq.Next(1)
+		require.NoError(err)
+		require.Equal(isequencer.Number(1), num)
+
+		seq.Flush()
+
+		storage.AddPLogEntry(initialOffset, 1, 1, num)
+
+		offset := isequencer.WaitForStart(t, seq, 1, 1, true)
+		require.Equal(isequencer.PLogOffset(2), offset)
+
+		num, err = seq.Next(1)
+		require.NoError(err)
+		require.Equal(isequencer.Number(2), num)
+
+		// Actualize with empty PLog
+		seq.Actualize()
+
+		// Should be able to start a new transaction
+		nextOffset := isequencer.WaitForStart(t, seq, 1, 1, true)
+		require.Equal(initialOffset+1, nextOffset)
+
+		num, err = seq.Next(1)
+		require.NoError(err)
+		require.Equal(isequencer.Number(2), num)
+	})
+
 	t.Run("empty plog", func(t *testing.T) {
 		storage := createDefaultStorage()
 		storage.SetPLog(map[isequencer.PLogOffset][]isequencer.SeqValue{})

